pkg/match/path.go

```diff
@@ -36,7 +43,7 @@ func getPathWords(path string) []string {
 	}
 
 	// handle path separators
-	const separator = `(?:_|[^\w\d])+`
+	const separator = `(?:_|[^\p{L}\w\d])+`
```

Comment:
It probably won't give a speed increase but I think the \w can be removed since \p{L} and \d already encompass it.  Same on line 84 I think.